algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        # Calculate phase difference
</pre><p>I don't understand this measure. Phase difference is acceptable as QC is always up to a global phase</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>